applications/StructuralMechanicsApplication/custom_elements/nodal_concentrated_element.cpp

```diff
@@ -393,21 +364,17 @@ void NodalConcentratedElement::CalculateMassMatrix( MatrixType& rMassMatrix, Pro
 
     //lumped
     unsigned int dimension = GetGeometry().WorkingSpaceDimension();
-    unsigned int SystemSize = dimension;
+    unsigned int system_size = dimension;
 
-    if ( rMassMatrix.size1() != SystemSize )
-    {
-        rMassMatrix.resize( SystemSize, SystemSize, false );
-    }
+    if ( rMassMatrix.size1() != system_size )
+        rMassMatrix.resize( system_size, system_size, false );
 
-    rMassMatrix = ZeroMatrix( SystemSize, SystemSize );
+    rMassMatrix = ZeroMatrix( system_size, system_size );
 
-    double &Nodal_Mass = Element::GetValue(NODAL_MASS);
+    const double nodal_mass = this->GetValue(NODAL_MASS);
```

Comment:
note that putting "const double"  is not enough. you need to make const the reference you call it onto.

that is you need to do. Doing this the NODAL_MASS would still be allocated
    
      const auto& rmyself = *this;
      rmyself.GetValue(...)